projects/packages/jetpack-mu-wpcom/src/features/launchpad/launchpad.php

```diff
@@ -576,6 +588,30 @@ function wpcom_launchpad_is_keep_building_enabled() {
 	return false;
 }
 
+/**
+ * Checks if the Blog flow task list is enabled.
+ *
+ * @return bool True if the task list is enabled, false otherwise.
+ */
+function wpcom_launchpad_is_intent_write_enabled() {
+
+	if ( ! is_automattician() ) {
+		return false;
+	}
+
+	$intent                  = get_option( 'site_intent', false );
+	$launchpad_task_statuses = get_option( 'launchpad_checklist_tasks_statuses', array() );
+
+	$launched = isset( $launchpad_task_statuses['site_launched'] ) && $launchpad_task_statuses['site_launched'];
+	$blog_id  = get_current_blog_id();
+
+	if ( 'write' === $intent && $blog_id > 220443356 && $launched ) {
```

Comment:
The site ID `220443356` was created June 28; we probably want to update this blog ID to a much more recent site ID before we launch. I'd add a TODO to that effect here.